packages/components/package.json

```diff
@@ -46,6 +48,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.6.4",
+    "@chanzuckerberg/axe-storybook": "^0.2.0",
```

Comment:
This uses playwright, instead of puppeteer, so we can run the tests in different browsers. Just noticed, though, that it doesn't install successfully on Node 8.

Can we drop Node v8 support from this project?